cli_tools_e2e_test/common/utils/utils.go

```diff
@@ -139,6 +139,15 @@ func RunTestCommand(cmd string, args []string, logger *log.Logger, testCase *jun
 	return true
 }
 
+// RunTestCommandIgnoringError runs given test command. The test case won't be marked as fail even error happens.
+func RunTestCommandIgnoringError(cmd string, args []string, logger *log.Logger, testCase *junitxml.TestCase) bool {
```

Comment:
```suggestion
// RunTestCommandExpectingError runs given test command. Passes when the command fails.
func RunTestCommandExpectingError(cmd string, args []string, logger *log.Logger, testCase *junitxml.TestCase) bool {
```

Reply:
I don't always expect error, instead, it can be either success or fail.